### PR TITLE
adding `dolt_conflicts_resolve` to stored procedures

### PR DESCRIPTION
**Pull request by jycor**

### CODE REVIEW

File: content/reference/sql/version-control/dolt-sql-procedures.md [hunk 1]
```diff
@@ -419,6 +420,41 @@ CALL DOLT_COMMIT('-a', '-m', 'This is a commit', '--author', 'John Doe <johndoe@
...
+## `DOLT_CONFLICTS_RESOLVE()`
+
+When a merge finds conflicting changes, it documents them in the dolt_conflicts table.
+A conflict is between two versions: ours (the rows at the destination branch head) and theirs
+(the rows at the source branch head).
+`dolt conflicts resolve` will automatically resolve the conflicts by taking either
+the ours or theirs versions for each row.
+
+```sql
+CALL DOLT_COMMIT('--ours', <table>);
```
Comment (timsehn) [inline content/reference/sql/version-control/dolt-sql-procedures.md]:
```suggestion
CALL DOLT_CONFICTS_RESOLVE('--ours', <table>);
```

File: content/reference/sql/version-control/dolt-sql-procedures.md [hunk 2]
```diff
@@ -419,6 +420,41 @@ CALL DOLT_COMMIT('-a', '-m', 'This is a commit', '--author', 'John Doe <johndoe@
...
+CALL DOLT_COMMIT('--theirs', <table>);
```
Comment (timsehn) [inline content/reference/sql/version-control/dolt-sql-procedures.md]:
```suggestion
CALL DOLT_CONFICTS_RESOLVE('--theirs', <table>);
```

**Review by timsehn — CHANGES_REQUESTED** [review]
Copy/pasta error i think.

File: content/reference/sql/version-control/dolt-sql-procedures.md [hunk 3]
```diff
@@ -419,6 +420,41 @@ CALL DOLT_COMMIT('-a', '-m', 'This is a commit', '--author', 'John Doe <johndoe@
...
+`--theirs`: For all conflicts, take the version from their branch and resolve the conflict.
+
+### Examples
+
+```sql
+-- Set the current database for the session
+USE mydb;
+
+-- Attempt merge
+CALL DOLT_MERGE('feature-branch');
```
Comment (zachmu) [inline content/reference/sql/version-control/dolt-sql-procedures.md:451]: Might have one additional step under this, where you do a select to see if there are conflicts

**Review by zachmu — APPROVED** [review]
LGTM, just one suggestion

**Pull request merged**